src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
Don`t create object inside this method, every invoke of method will create new object but you need only one